app.py

```diff
@@ -50,7 +60,14 @@ def process_import_function(event, context):
     unique_jurisdictions = {}
 
     # Get the uploaded file's information
-    messages = batch_retrieval_from_sqs()
+    sqs_fetch_batch_size = int(os.environ.get("SQS_FETCH_BATCH_SIZE", 600))
+    sqs_delete_fetched_messages = os.environ.get("SQS_DELETE_FETCHED_MESSAGES", True)
+    if (
+            sqs_delete_fetched_messages is not False
+            and sqs_delete_fetched_messages.lower() != "true" and sqs_delete_fetched_messages != "1"
```

Comment:
I don't follow this logic. Doesn't `sqs_delete_fetched_messages.lower() != "true" and sqs_delete_fetched_messages != "1"` mean that `sqs_delete_fetched_messages` is probably "false" or "0". Shouldn't that be `sqs_delete_fetched_messages = False` then?

Reply:
You're right, that was some horrible code. Replacing with a library to avoid logic errors